.prettierignore

```diff
@@ -3,9 +3,3 @@ node_modules
 /build
 /public/build
 .env
-
-/cypress/screenshots
-/cypress/videos
-/postgres-data
```

Comment:
Add `/mysql-data` so it doesn't run prettier in there (that's the new database volume).

Reply:
^^